.gitignore

```diff
@@ -8,3 +8,5 @@ lerna-debug.log
 **/.DS_Store
 **/*.env*
 !**/.env.schema
+
+.ignore/
```

Comment:
does what it says on the tin

Reply:
Is this something you think the project needs due to an expected folder that our code generates? Or are you using the committed .gitignore to ignore files in your own local setup? If the latter, may I suggest a local file in your .git directory: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file